partials/algolia-search.hbs

```diff
@@ -50,9 +51,9 @@
           if (!query.isEmpty) {
             return `
               <div class="aa-suggestion footer-suggestion">
-                <a id="algolia-footer-selector" href="{{@site.url}}/search?query=${result.query}">
+                <a id="algolia-footer-selector" href="{{@site.url}}/search?query=${searchQuery}">
```

Comment:
Possibly a silly question, but what happens if I copy and paste my query into the search box?  We don't listen to mouse events, so is it possible the `searchQuery` would still be `undefined`?

Reply:
Sorry @ojeytonwilliams, just seeing this now.

Not a silly question at all! This is something I totally missed in my testing.

Right now if you just paste a search term into News without touching the keyboard, results will still show if there's a match, but there won't be anything in the search footer.

I'll play around with this a bit more and work on a fix.